.vscode/extensions.json

```diff
@@ -69,7 +69,7 @@
 
     // Name: Toggle Quotes
 		// Description: Toggle cycle " -> ' -> `
```

Comment:
Not this PR's fault, but it looks like I messed up this file by mixing `\t` and spaces. I can submit a PR to fix this after your Pr merges if you want to keep this PR isolated to changes that only exist in 10.0-release.